src/js/components/PodInstancesTable.js

```diff
@@ -255,14 +258,13 @@ class PodInstancesTable extends React.Component {
   }
 
   renderColumnAddress(prop, row, rowOptions = {}) {
-    if (rowOptions.isParent) {
-      return this.renderWithClickHandler(rowOptions, (
-        <CollapsingString string={row.address} />
-      ));
+    if (!rowOptions.isParent) {
```

Comment:
Yes, that's correct, because parent renders the host address (`row.address` is a string) while the children render an array of ports (`row.address` is an array of nodes).